graphene_django/utils/utils.py

```diff
@@ -12,6 +16,28 @@
     DJANGO_FILTER_INSTALLED = False
 
 
+def isiterable(value):
+    try:
+        iter(value)
+    except TypeError:
+        return False
+    return True
+
+
+def _camelize_django_str(s):
+    if isinstance(s, Promise):
+        s = force_text(s)
+    return to_camel_case(s) if isinstance(s, six.string_types) else s
+
+
+def camelize(data):
```

Comment:
Is this function the same as `graphene.utils.str_converters.to_camel_case` ?
https://github.com/graphql-python/graphene/blob/ac6714e8fa61d3cb6d5f73b2614ce65b3cb13941/graphene/utils/str_converters.py#L6

Reply:
Nvm, I see it now.